docs/UserGuide.md

```diff
@@ -47,140 +44,200 @@ Format: `help`
+Format: `add t/TASK [d/DATE] [#tags]`
 
 > Persons can have any number of tags (including 0)
 
 Examples:
 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe t/friend e/betsycrowe@gmail.com a/Newgate Prison p/1234567 t/criminal`
+* `add t/CS2103 meeting d/03/03 #Programming Lab 2`
 
-### 2.3. Listing all persons : `list`
+### 2.3. Listing all tasks : `list`
 
-Shows a list of all persons in the address book.<br>
+Shows a list of all tasks in the task list.<br>
 Format: `list`
 
-### 2.4. Editing a person : `edit`
+### 2.4. Create a list: `createlist`
+
+Create a new list. <br>
+Format: `createlist NAME`
+
+The name refers to name of the task list.
+
+Examples:
+* `createlist Sports` <br>
+  Create a new list named Sports.
+
+### 2.5. Editing a person : `edit`
 
 Edits an existing person in the address book.<br>
-Format: `edit INDEX [NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
+Format: `edit INDEX [t/TASK] [d/DATE] [#tags]`
 
 > * Edits the person at the specified `INDEX`.
     The index refers to the index number shown in the last person listing.<br>
     The index **must be a positive integer** 1, 2, 3, ...
 > * At least one of the optional fields must be provided.
 > * Existing values will be updated to the input values.
 > * When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-> * You can remove all the person's tags by typing `t/` without specifying any tags after it. 
+> * You can remove all the person's tags by typing `#` without specifying any tags after it. 
 
 Examples:
 
-* `edit 1 p/91234567 e/johndoe@yahoo.com`<br>
-  Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@yahoo.com` respectively.
+* `edit 1  d/22/03`<br>
+  Edits the due date of the `1st task` to be `22/03`.
 
-* `edit 2 Betsy Crower t/`<br>
-  Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+* `edit 2 t/Project meeting #iCube`<br>
+  Edits the `2nd task` to Project meeting and add hashtag `iCube..`.
 
-### 2.5. Finding all persons containing any keyword in their name: `find`
+* `edit 2 t/Project meeting #`<br>
+  Edits the `2nd task` to Project meeting and clear all existing tags.
+  
+### 2.6. Finding all tasks containing any keyword in their title or due dates: `find`
 
-Finds persons whose names contain any of the given keywords.<br>
+Finds tasks containing any of the given keywords or due in given date.<br>
 Format: `find KEYWORD [MORE_KEYWORDS]`
 
-> * The search is case sensitive. e.g `hans` will not match `Hans`
-> * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-> * Only the name is searched.
-> * Only full words will be matched e.g. `Han` will not match `Hans`
-> * Persons matching at least one keyword will be returned (i.e. `OR` search).
-    e.g. `Hans` will match `Hans Bo`
+> * The search is case sensitive. e.g `project` will not match `Project`
```

Comment:
think if you can remove this restriction! that will make your product more user friendly